myRBM/rbm.asv

```diff
@@ -0,0 +1,61 @@
+%% load MNIST dataset (see RBM)
```

Comment:
You can add .asv and .DS_Store binary files to your gitignore file, that way it doesn't get pushed to github. Will help keep things clean